src/platformIcon.tsx

```diff
@@ -12,16 +11,21 @@ const PLATFORM_TO_ICON = {
   csharp: "csharp",
   "csharp-aspnetcore": "dotnet",
   dart: "dart",
+  "dart-flutter": "flutter",
   default: "default",
   dotnet: "dotnet",
   "dotnet-aspnetcore": "dotnet",
+  "dotnet-aspnet": "dotnet",
+  "dotnet-csharp": "csharp",
+  "dotnet-fsharp": "csharp",
+  "dotnet-blazor": "blazor",
+  "dotnet-uno": "uno",
+  "dotnet-xamarin": "xamarin",
   electron: "electron",
   elixir: "elixir",
-  flutter: "flutter",
   go: "go",
   godot: "godot",
   java: "java",
-  "java-android": "android",
```

Comment:
The preferred language for Android development for a few years is Kotlin.
For that reason removing this Java Android. Does this break something?